chef/cookbooks/neutron/templates/default/ovs-usurp-config.erb

```diff
@@ -18,6 +18,11 @@ rc_reset
 
 case "$1" in
     start)
+        # (aplanas) temporal workaround for bsc#930986 proposed by
+        # rhafer. A better workaround based on
+        # https://en.opensuse.org/Portal:Wicked/OpenvSwitch is in
+        # planning.
+        ip link set up <%= @source %>
```

Comment:
Does this really work here? I expected it to be needed after the flush calls. Or even as the very last step of "start"